Automattic/jetpack
# WooCommerce Anaylytics: PHPCS

WPCS for the woocommerce-analytics module.
Fixes #10117
Closes #10496

#### Changes proposed in this Pull Request:
* PHPCS

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* n/a

#### Does this pull request change what data or activity we track or use?


* No changes.

#### Testing instructions:






* Connected JP + WC
* See testing instructions in #15127 to ensure everything still works.
* Note you should try when not logged in as an admin on the site.

#### Proposed changelog entry for your changes:

* n/a
